oak/server/rust/oak_runtime/src/metrics/server.rs

```diff
@@ -42,9 +43,11 @@ impl std::fmt::Display for MetricsServerError {
 
 impl std::error::Error for MetricsServerError {}
 
-async fn serve_metrics(_req: Request<Body>) -> Result<Response<Body>, MetricsServerError> {
+async fn handle_metrics_request(
+    runtime: Arc<Runtime>,
```

Comment:
Why does this need to be an `Arc`? I thought it should be enough to have a regular reference. Or does that complicate the lifetime of this method?